### PR TITLE
Add BlueBEAR cluster config

### PR DESCRIPTION
**Pull request by alexlyttle**
---
name: BlueBEAR
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: conf/bluebear.config
Comment (jfy133) [inline conf/bluebear.config:18]: Having non-pipeline parameters here may fail some (and increasingly more pipelines).

A better approach is to tell users to set a bash environment variable with the information needed, and yse that information (in this case down in your clusterOptions).

For example, see https://github.com/nf-core/configs/blob/2900904b814873e911218c60972c15db7e119341/conf/pawsey_setonix.config#L26

File: docs/bluebear.md
Comment (jfy133) [inline docs/bluebear.md:19]: Maybe mention it's a good idea to put the `export` command in a `.bashrc` or similar?

File: docs/bluebear.md
Comment (jfy133) [inline docs/bluebear.md:82]: You could consider by default adding the `cleanup = true` [^1] option to have the emptying of the `work/` happen by default, and to override this use the `debug` profile in addition to the institutional profile.

For examples: https://github.com/search?q=repo%3Anf-core%2Fconfigs%20cleanup%20%3D%20true&type=code

[^1]: Docs: https://www.nextflow.io/docs/latest/config.html#miscellaneous

**Review by jfy133 — COMMENTED** [review]
A couple of minor comments, if you disagree we can proceed.

Note, so you don't ave to wait for approval for the tests to run - consider joining the nf-core github organisation, instructions here: https://nf-co.re/join

File: conf/bluebear.config
Comment (alexlyttle) [reply conf/bluebear.config:18]: Thanks, that sounds good. In fact `System.getenv("SLURM_JOB_ACCOUNT")` could be substituted there, since it is unlikely someone would want individual processes to be submitted under a different account.

File: docs/bluebear.md
Comment (alexlyttle) [reply docs/bluebear.md:82]: I was a little concerned about the warning in the Nextflow docs about this. It suggests `cleanup = true` prevents use of `-resume`. If cleanup only occurs on a successful run, is `-resume` still possible after a failed run?

Comment (jfy133) [reply docs/bluebear.md:82]: It only prevents `-resume` if the pipeline completes with exit code 0 (so no errors).

If the pipeline hits an error (exit code >1), all the files in `work` are retained and will not be deleted

The only case that it might be problematic is if someone is playing with parameters in later stages of a pipeline, and want to resume from midway but with a different parameter, yes in this case this `resume` would not work (if that makes sense)

Comment (alexlyttle) [reply docs/bluebear.md:82]: Ah fantastic, I misunderstood the docs slightly. This sounds like a good addition, thanks!

File: docs/bluebear.md
Comment (alexlyttle) [reply docs/bluebear.md]: Done!

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
Good to go @alexlyttle !

With your new found nf-core org membership, you may do the honours! (At least I think you should be able to merge yourself now 😅)

**Pull request merged**
